### DISCUSSION TITLE
Markdown 增强中的 Mermaid 能配置图表颜色吗？

### DISCUSSION THREAD

**Discussion by llnancy** [vuepress-theme-hope/vuepress-theme-hope#3046] [Q&A] [answered]
目前只看到一个启用配置项：

```ts
// 启用 mermaid
mermaid: true,
```

想改变图表颜色是只能自己写 css 覆盖吗？

**Comment by Mister-Hope** [comment]
https://plugin-md-enhance.vuejs.press/config.html#mermaid

**Comment by Mister-Hope** [comment, marked as answer]
You can now use `defineMermaidConfig`

**Reply by clay-wangzhi** [reply]
I use defineMermaidConfig but it didn't take effect.
like this
```
      mermaid: {
        themeVariables: {
          mainBkg: "#e9e9fe",
          nodeBorder: "#9672d6"
        }
``` 
In the end, I managed to make it work by initializing it within the markdown.
like this
```
%%{
  init: {
    'themeVariables': {
      'mainBkg': '#e9e9fe',
      'nodeBorder': '#9672d6'
    }
  }
}%%
```

**Reply by Mister-Hope** [reply]
> I don't think you need that outer mermaid key

Mermaid never requires you to set options under mermaid key

Try to understand what others are talking.

**Reply by clay-wangzhi** [reply]
No, I need a global configuration so that I don't have to configure each mermaid individually.

**Reply by Mister-Hope** [reply]
No further reply, if you think something is not working, provide me a minimal reproduction in a new issue.
